bar-trender-front-end/src/components/ModalSelectedDiscount.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from "react";
+import {
+  Row,
+  Col,
+} from "reactstrap";
+
+// reactstrap components
+import {
+  Button,
+  Modal,
+  ModalBody,
+  ModalFooter,
+} from "reactstrap";
+
+// core components
+import image_left from '../assets/img/bg-landing.png';
+import Qr from '../App.js'
+
+//css components
+import '../views/css/ModalQR.css'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7138516)
```suggestion
import "../views/css/ModalQR.css"
```